MekHQ/src/mekhq/campaign/parts/Refit.java

```diff
@@ -848,7 +848,9 @@ public void calculate() {
         for (AmmoType type : ammoNeeded.keySet()) {
             int shotsNeeded = Math.max(ammoNeeded.get(type) - getAmmoAvailable(type), 0);
             int shotsPerTon = type.getShots();
-            cost = cost.plus(Money.of(type.getCost(newEntity, false, -1) * ((double)shotsNeeded/shotsPerTon)));
+            if (shotsNeeded > 0 && shotsPerTon > 0) {
```

Comment:
```suggestion
            if ((shotsNeeded > 0) && (shotsPerTon > 0)) {
```

Reply:
My IDE reports those as extraneous parentheses and removes them on save (I wouldn't add them either given the simplicity of the line).